test/test.js

```diff
@@ -1,7 +1,6 @@
-import test from 'blue-tape'
-import db from '../src'
-import pg from 'pg'
-import 'babel-polyfill'
+const test = require('blue-tape')
```

Comment:
Removed babel dependency in the test suite because async/await is supported in recent nodejs natively. Switched from `import` to `require` to facilitate that.